opensearch-project/OpenSearch-Dashboards
# [Workspace]Fix error message missed in workspace creator name input

### Description

This PR is for fixing error message missed in workspace creator name and color input.

## Screenshot


![image](https://github.com/user-attachments/assets/764d1676-5eac-4d39-a7c8-dbce6a491de9)


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version loose`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Login with admin user and go to workspace create page by bottom left menu
- Click the workspace create button and the name input error should be displayed

## Changelog

-fix: [Workspace]Fix error message missed in workspace creator name input

### Check List
